var/spack/repos/builtin/packages/py-virtualenv/package.py

```diff
@@ -21,6 +22,18 @@ class PyVirtualenv(PythonPackage):
     version('1.11.6', sha256='3e7a4c151e2ee97f51db0215bfd2a073b04a91e9786df6cb67c916f16abe04f7')
 
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@20.10.0:')
 
     # not just build-time, requires pkg_resources
     depends_on('py-setuptools@40.6.3:', type=('build', 'run'))
+    depends_on('py-setuptools@41.00.03:', type=('build', 'run'), when='@20.10.0:')
+    depends_on('py-setuptools-scm@2:', type=('build', 'run'), when='@20.10.0:')
+
+    depends_on('py-backports-entry-points-selectable @1.0.4:', type=('build', 'run'), when='@20.10.0:')
+    depends_on('py-distlib@0.3.1:0', type=('build', 'run'), when='@20.10.0:')
```

Comment:
Apologies I just noticed this has an upper bound of `0` - was that intended?

Reply:
@scheibelp yes, it needs any version below 1.